test/resampling.jl

```diff
@@ -801,8 +802,23 @@ end
     measures = [LogLoss(), Accuracy()]
     evaluations = evaluate(clf, X, y; measures, resampling=Holdout())
     show_text = sprint(show, MIME"text/plain"(), evaluations)
-    print(show_text)
     @test !contains(show_text, "std")
+
+    # issue #871: trying to calculate SE when inappropriate should not throw an error in
+    # display.
+    evaluations = evaluate(
+        clf,
+        X,
+        y,
+        operation=predict_mode,
+        measure=ConfusionMatrix(),
+        resampling=CV(),
+    )
+    io = IOBuffer()
+    show(io, evaluations)
+    show(io, MIME("text/plain"))
+    confusion_table = sprint(show, "text/plain", evaluations.measurement[1])
+    @test contains(confusion_table, "Ground Truth")
```

Comment:
```suggestion
    printed_evaluations = sprint(show, "text/plain", evaluations)
    @test contains(printed_evaluations, "N/A")
```

This catches the failure reported in #871. My `sprint(show, "text/plain", evaluations.measurement[1])` test didn't so can be removed.

Reply:
With old code in `src/resampling` (without this PR):
```julia
julia> sprint(show, "text/plain", evaluations)
ERROR: MethodError: no method matching /(::MLJBase.ConfusionMatrixObject{2}, ::Int64)
[...]
```
With new code in `src/resampling` (with this PR):
```julia
julia> sprint(show, "text/plain", evaluations) |> print
PerformanceEvaluation object with these fields:
  measure, operation, measurement, per_fold,
  per_observation, fitted_params_per_fold,
  report_per_fold, train_test_rows
Extract:
┌───────────────────┬──────────────┬──────────────────────────────────────────────────────┬─────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ measure           │ operation    │ measurement                                          │ 1.96*SE │ per_fold                                                                                                                                                                                                                                                                                                                                           │
├───────────────────┼──────────────┼──────────────────────────────────────────────────────┼─────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ ConfusionMatrix(  │ predict_mode │ ConfusionMatrixObject{2}([18 32; 32 18], ["0", "1"]) │ N/A     │ MLJBase.ConfusionMatrixObject{2}[ConfusionMatrixObject{2}([6 11; 0 0], ["0", "1"]), ConfusionMatrixObject{2}([0 0; 10 7], ["0", "1"]), ConfusionMatrixObject{2}([7 10; 0 0], ["0", "1"]), ConfusionMatrixObject{2}([0 0; 10 7], ["0", "1"]), ConfusionMatrixObject{2}([5 11; 0 0], ["0", "1"]), ConfusionMatrixObject{2}([0 0; 12 4], ["0", "1"])] │
│   perm = nothing) │              │                                                      │         │                                                                                                                                                                                                                                                                                                                                                    │
└───────────────────┴──────────────┴──────────────────────────────────────────────────────┴─────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Reply:
I stand corrected. Thanks for the review and taking the time to sort out my misunderstanding.